tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -729,21 +729,6 @@ func (s *storeImpl) GetIDs(ctx context.Context) ([]{{$singlePK.Type}}, error) {
 }
 {{- end }}
 
-{{- if .GetAll }}
```

Comment:
If not already done in the cleanup PR, make sure the get-all option is removed from the generator code as well as from the gen.go files that use it.

Reply:
The complete cleanup would impact:
- the gen.go files next to the stores generated by this pull request
- tools/generate-helpers/pg-table-bindings/migration.go.tpl (or the file could be removed completely, as well as the gen.go file in the migrator)
- tools/generate-helpers/pg-table-bindings/store.go.tpl (but I think the current pull request already removes all occurrences)
- tools/generate-helpers/pg-table-bindings/store_test.go.tpl (to let all stores test the GetAll function)
- tools/generate-helpers/pg-table-bindings/main.go

Reply:
So you think we should expose `GetAll` method to all stores? I wander what was the reason to have it as optin

Reply:
https://issues.redhat.com/browse/ROX-18312